trunk/SUAVE/Methods/Propulsion/ramjet_sizing.py

```diff
@@ -0,0 +1,167 @@
+# ramjet_sizing.py
+# 
+# Created:  May 2015, T. MacDonald 
+# Modified: Jan 2016, E. Botero        
+
+# ----------------------------------------------------------------------
+#   Imports
+# ----------------------------------------------------------------------
+
+import SUAVE
+import numpy as np
+from SUAVE.Core import Data
+
+# ----------------------------------------------------------------------
+#   Sizing
+# ----------------------------------------------------------------------
+
+def ramjet_sizing(ramjet,mach_number = None, altitude = None, delta_isa = 0, conditions = None):  
+    """ create and evaluate a ramjet etwork
+    """    
+    
+    #Unpack components
+    #check if altitude is passed or conditions is passed
+    if(conditions):
+        #use conditions
+        pass
+        
+    else:
+        #check if mach number and temperature are passed
+        if(mach_number==None or altitude==None):
+            
+            #raise an error
+            raise NameError('The sizing conditions require an altitude and a Mach number')
+        
+        else:
+            #call the atmospheric model to get the conditions at the specified altitude
+            atmosphere = SUAVE.Analyses.Atmospheric.US_Standard_1976()
+            atmo_data = atmosphere.compute_values(altitude,delta_isa)
+
+            p   = atmo_data.pressure          
+            T   = atmo_data.temperature       
+            rho = atmo_data.density          
+            a   = atmo_data.speed_of_sound    
+            mu  = atmo_data.dynamic_viscosity   
+        
+            # setup conditions
+            conditions = SUAVE.Analyses.Mission.Segments.Conditions.Aerodynamics()            
+        
+            # freestream conditions    
+            conditions.freestream.altitude           = np.atleast_1d(altitude)
+            conditions.freestream.mach_number        = np.atleast_1d(mach_number)
+            conditions.freestream.pressure           = np.atleast_1d(p)
+            conditions.freestream.temperature        = np.atleast_1d(T)
+            conditions.freestream.density            = np.atleast_1d(rho)
+            conditions.freestream.dynamic_viscosity  = np.atleast_1d(mu)
+            conditions.freestream.gravity            = np.atleast_1d(9.81)
```

Comment:
Maybe these aren't practical to change right now, but these values shouldn't be hard coded.